legacy-usage.md

```diff
@@ -0,0 +1,448 @@
+```
+$ java -Dcom.signalfx.public.java.disableHttpCompression=true ...
+```
+
+## Example Project
+
+You can find a full-stack example project called "signalfx-java-examples" in
+the repo.
+
+Run it as follows:
+
+1. Download the code and create an "auth" file in the "signalfx-java-examples"
+   directory. The auth file should contain the following:
+
+    ```
+    auth=<signalfx API Token>
+    host=https://ingest.signalfx.com
+    ```
+
+2. Run the following commands in your terminal to install and run the example
+   project, replacing `path/to/signalfx-java-examples` with the location of the
+   example project code in your environment. You must have Maven installed.
+
+    ```
+    cd path/to/signalfx-java-examples
+    mvn install
+    # an example for Yammer 2.x metrics
+    mvn exec:java -Dexec.mainClass="com.signalfx.example.YammerExample"
+    # an example for sending datapoints and events using protocol buffers
+    mvn exec:java -Dexec.mainClass="com.signalfx.example.ProtobufExample"
+    ```
+New metrics and events from the example project should appear in SignalFx.
+
+## Executing SignalFlow computations
+
+SignalFlow is SignalFx's real-time analytics computation language. The
+SignalFlow API allows SignalFx users to execute real-time streaming analytics
+computations on the SignalFx platform. For more information, head over to our
+Developers documentation:
+
+* [SignalFlow Overview](https://developers.signalfx.com/signalflow_analytics/signalflow_overview.html)
+* [SignalFlow API Reference](https://developers.signalfx.com/signalflow_reference.html)
+
+Executing a SignalFlow program is very simple with this client library:
+
+```java
+String program = "data('cpu.utilization').mean().publish()";
+SignalFlowClient flow = new SignalFlowClient("MY_TOKEN");
+System.out.println("Executing " + program);
+Computation computation = flow.execute(program);
+for (ChannelMessage message : computation) {
+    switch (message.getType()) {
+    case DATA_MESSAGE:
+        DataMessage dataMessage = (DataMessage) message;
+        System.out.printf("%d: %s%n",
+                dataMessage.getLogicalTimestampMs(), dataMessage.getData());
+        break;
+
+    case EVENT_MESSAGE:
+        EventMessage eventMessage = (EventMessage) message;
+        System.out.printf("%d: %s%n",
+                eventMessage.getTimestampMs(),
+                eventMessage.getProperties());
+        break;
+    }
+}
+```
+
+Metadata about the timeseries is received from the iterable stream, and it
+is also automatically intercepted by the client library and made available through
+the ``Computation`` object returned by ``execute()``:
+
+```java
+case DATA_MESSAGE:
+    DataMessage dataMessage = (DataMessage) message;
+    for (Map<String, Number> datum : dataMessage.getData()) {
+        Map<String,Object> metadata = computation.getMetadata(datum.getKey());
+        // ...
+    }
+```
+
+## License
+
+Apache Software License v2. Copyright © 2014-2023 SignalFx | Splunk
```

Comment:
Is this part needed in this doc? We have that in the readme

Reply:
I'll remove it.